assets/javascripts/discourse/templates/components/tds-new-user.hbs

```diff
@@ -0,0 +1,40 @@
+<tr class="input">
+  <td></td>
+  <td class="collective-selectors">
+    <!-- one category per collective -->
+    {{#each site.categories as |c|}}
```

Comment:
We probably need to filter these a little further, right? Idk if we can be certain the the _only_ categories will be the collective ones.

Reply:
Ah, I thought that was a guarantee. I'll look into how to do that.

Reply:
There might be ones for the dispute admins to talk and stuff like that? I feel like we may have mentioned something along those lines.